cosmos/templates/index.html

```diff
@@ -1 +1,156 @@
-{% extends "base.html" %}
\ No newline at end of file
+{% extends "base.html" %} {% load static %} {% block base-content %}
+<section class="home-img min-vh-100">
+    <div class="container d-flex h-100">
+        <div class="mx-auto text-center">
+            <img class="img-fluid mh-50" src="{% static 'cosmos/img/cosmos_white_horizontal.png' %}">
+        </div>
+    </div>
+</section>
+<section id="stats" class="section-bg">
+    <div class="container py-5">
+        <div class="row">
+            <div class="col-lg-3 col-6">
+                <div class="stat-box">
+                    <span>69</span>
```

Comment:
should we add the functions to count the number of nationalities in the same PR or separate PR (likewise for the no of members, active years, #events per year) @bcvandendool @maximekabel

Reply:
I think we can do that in a separate pr, cause rn the branches are getting too out of sync, so i want these pull requests to get merged asap

Reply:
I think we should add it later, I don't think it's useful to spend time on smth like this rn